packages/mobile/src/fees/CalculateFee.tsx

```diff
@@ -0,0 +1,133 @@
+import { getStableTokenContract } from '@celo/contractkit'
+import BigNumber from 'bignumber.js'
+import React, { FunctionComponent, useEffect } from 'react'
+import { useAsync, UseAsyncReturn } from 'react-async-hook'
+import { connect } from 'react-redux'
+import { showError } from 'src/alert/actions'
+import { ErrorMessages } from 'src/app/ErrorMessages'
+import { ERROR_BANNER_DURATION } from 'src/config'
+import { getReclaimEscrowFee } from 'src/escrow/saga'
+import { FeeType } from 'src/fees/actions'
+import { getInvitationVerificationFee } from 'src/invite/saga'
+import { getSendFee } from 'src/send/saga'
+
+export type CalculateFeeChildren = (asyncResult: UseAsyncReturn<BigNumber, never>) => any
```

Comment:
Will `React.ReactNode` work here instead of any?

Reply:
Let me try.

Reply:
Here it is: https://github.com/celo-org/celo-monorepo/commit/fcbc7ab871bf9f43876722957006fa311f4188f6
It caused another type issue, which I worked around with a cast.
Anyway definitely better with `ReactNode` 👍